### ISSUE TITLE
Urgent!!!! The Redux 3.6 Framework plugin takes control of the admin plugins link!

### ISSUE THREAD

**Issue by a4jp-com** [reduxframework/redux-framework#2934]
The Redux 3.6 Framework plugin takes control of the admin plugin link.

It advertises itself and hides all the other plugins in WordPress 4.5!

http://support.redux.io/?id=WsuE7S%2FL9zp0xyPsDKRwA4hOhT%2Fq9%2Bufsx%2FazOk2jlmz8Z3gq13%2BemvDyHHMf8E4ve4QCWaV6nkdcyZ8YupgDg%3D%3D

**Comment by kprovance** [comment]
Are you able to give us WP admin and server access?  We are not able to replicate this here and need to be in an environment where this is an issue.

**Comment by a4jp-com** [comment]
Very good news. I found a test theme I downloaded from a site had some nasty code it in. It activated itself when your update came out at the same time and somehow linked your page to the plugin button. Everything is back to normal now. Sorry for the alarm I just thought because the update was the only change I made before your plugin started advertising itself that the Redux Framework caused the problem.
